app/components/TaskList.tsx

```diff
@@ -156,36 +127,13 @@ const TaskList: React.FC<Props> = ({ projectRevision }) => {
           margin: 0;
         }
 
-        ul {
-          list-style: none;
-          margin: 0;
-        }
-
-        li {
-          text-indent: 15px;
-          margin-bottom: 0;
-        }
-
-        ul > li {
-          display: flex;
-          justify-content: space-between;
-        }
-
         h2 {
           font-size: 1.25rem;
           margin: 0;
           padding: 20px 0 10px 0;
           border-bottom: 1px solid #d1d1d1;
           text-indent: 15px;
         }
-        h3 {
-          font-size: 1rem;
-          line-height: 1;
-          border-bottom: 1px solid #d1d1d1;
-          text-indent: 15px;
-          padding: 10px 0 10px 0;
-          margin: 0;
-        }
```

Comment:
I think the styles below for the links are not needed if we use the `Link` components from the BC gov theme.
See the `Dashboard` component, which uses the `BCGovLink` component